SocialCareCaseViewerApi.Tests/V1/Gateways/MashReferralGatewayTests/UpdateReferralTests.cs

```diff
@@ -185,5 +193,14 @@ public void SuccessfulUpdateOfMashReferralAssignsWorkerUsingEmailAndReturnsMashR
             response.Should().BeEquivalentTo(mashReferral.ToDomain());
         }
 
+        [Test]
+        public void SuccessfulUpdateOfMashReferralUnAssignsWorkerAndReturnsMashReferralDomain()
+        {
+            var mashReferral = MashReferralHelper.SaveMashReferralToDatabase(DatabaseContext);
+            var request = TestHelpers.CreateUpdateMashReferral(updateType: "UNASSIGN-WORKER");
+            var response = _mashReferralGateway.UpdateReferral(request, mashReferral.Id);
+
+            response.Should().BeEquivalentTo(mashReferral.ToDomain());
```

Comment:
Is there a way for us to check the value of the worker attribute as part of the assertion?
At a glance, I don't know if the test helper `SaveMashReferralToDatabase` adds a worker to a Mash Referral, could we check this as part of the test and then after the update request check for an expected change to that value?

Reply:
Added explicit set to the WorkerId before the update and checking it equals to null.